support/ppTiming.cpp

```diff
@@ -5,6 +5,7 @@
 #include <algorithm>
 #include <sstream>
 #include <iomanip>
+#include <cmath>
```

Comment:
we should note that this is for calling `trunc` and `log10`

```suggestion
#include <cmath> // trunc, log10
```